lib/flex_commerce_api/json_api_client_extension/included_data.rb

```diff
@@ -1,12 +1,26 @@
-module FlexCommerce
+require 'json_api_client/included_data'
+
+module FlexCommerceApi
   module JsonApiClientExtension
     class IncludedData < ::JsonApiClient::IncludedData
+      def initialize(result_set, *args)
+        super(result_set, *args)
+        @result_set = result_set
```

Comment:
This is obviously a workaround as you are going outside of the included datas responsibility of the included data class. As long as youre ok with this ?